benefit/data/sheets/RoleToAccessProfiles/RoleToAccessProfiles-nonprod.json

```diff
@@ -3,12 +3,22 @@
     "LiveFrom": "18/06/2019",
     "CaseTypeID": "Benefit",
     "RoleName": "idam:caseworker-sscs-dwpresponsewriter",
-    "CaseAccessCategory": "taxCredit,guardiansAllowance",
+    "CaseAccessCategories": "personalIndependencePayment,carer'sAllowance,jobseeker’sAllowance,employmentAndSupportAllowance,disabilityLivingAllowance,universalCredit,industrialDeathBenefit,pensionCredit,retirementPension,childSupport,bereavementBenefit,industrialInjuriesDisablement,maternityAllowance,socialFund,incomeSupport,bereavementSupportPaymentScheme,attendanceAllowance",
```

Comment:
Should these have apostrophes? Should they be the short names in https://github.com/hmcts/sscs-common/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java?